build.sbt

```diff
@@ -157,11 +126,6 @@ lazy val commonSettings = Seq(
   unidocSourceFilePatterns := Nil,
 )
 
-// enforce java code style
```

Comment:
There are many changes in this and other files that are certainly not required to support Hive 4 which must be removed

Reply:
You can't remove this checkstyle.